lte/gateway/python/magma/pipelined/app/inout.py

```diff
@@ -155,51 +154,34 @@ def _get_config(self, config_dict):
     def initialize_on_connect(self, datapath):
         self._datapath = datapath
         self._setup_non_nat_monitoring()
-        # TODO remove temp fix
-        # if self.config.setup_type == 'XWF':
-        self.delete_all_flows(datapath)
-        self._install_default_flows_if_not_installed(datapath, {})
-
-    def _install_default_flows_if_not_installed(self, datapath,
-            existing_flows: List[OFPFlowStats]) -> List[OFPFlowStats]:
-        ret = {}
-        ingress_msgs = self._get_default_ingress_flow_msgs(datapath)
-        current_flows = []
-        if self._ingress_tbl_num in existing_flows:
-            current_flows = existing_flows[self._ingress_tbl_num]
-        msgs, remaining_flows = self._msg_hub \
-            .filter_msgs_if_not_in_flow_list(self._datapath, ingress_msgs,
-                                             current_flows)
-        ret[self._ingress_tbl_num] = remaining_flows
-        if msgs:
-            chan = self._msg_hub.send(msgs, datapath)
-            self._wait_for_responses(chan, len(msgs))
-
-        egress_msgs = self._get_default_egress_flow_msgs(datapath)
-        current_flows = []
-        if self._egress_tbl_num in existing_flows:
-            current_flows = existing_flows[self._egress_tbl_num]
-        msgs, remaining_flows = self._msg_hub \
-            .filter_msgs_if_not_in_flow_list(self._datapath, egress_msgs,
-                                             current_flows)
-        ret[self._egress_tbl_num] = remaining_flows
-        if msgs:
-            chan = self._msg_hub.send(msgs, datapath)
-            self._wait_for_responses(chan, len(msgs))
-
-        middle_msgs = self._get_default_middle_flow_msgs(datapath)
-        current_flows = []
-        if self._midle_tbl_num in existing_flows:
-            current_flows = existing_flows[self._midle_tbl_num]
-        msgs, remaining_flows = self._msg_hub \
-            .filter_msgs_if_not_in_flow_list(self._datapath, middle_msgs,
-                                             current_flows)
-        ret[self._midle_tbl_num] = remaining_flows
-        if msgs:
-            chan = self._msg_hub.send(msgs, datapath)
-            self._wait_for_responses(chan, len(msgs))
-
-        return ret
+
+        if self.config.setup_type == 'XWF':
+            self.delete_all_flows(datapath)
+            self._install_default_flows(datapath)
```

Comment:
XWF flows are not handled by restart controller? can you mark it as todo.

Reply:
yeah XWF doesn't run sessiond and we don't currently support stateless without it. I'll add a TODO